player/command.c

```diff
@@ -2916,6 +2916,29 @@ static int mp_property_sub_pos(void *ctx, struct m_property *prop,
     return mp_property_generic_option(mpctx, prop, action, arg);
 }
 
+static int mp_property_sub_extradata(void *ctx, struct m_property *prop,
+                                     int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+    struct track *track = mpctx->current_track[0][STREAM_SUB];
```

Comment:
`current_track` can be null

Reply:
Isn't the next line sufficient? I checked other functions and that's how we do it everywhere.

```
    struct dec_sub *sub = track ? track->d_sub : NULL;
    if (!sub)
        return M_PROPERTY_UNAVAILABLE;
```

I also made sure to test with files with no subtitles, or files with subs but none selected and it returns property unavailable without exploding like it should.

Reply:
Ah you're right. I misunderstood the comment in `core.h`. The `current_track[MAX_PTRACKS][STREAM_TYPE_COUNT]` is what's NULL not `current_track`.